docs/src/domains/cdn/_index.md

```diff
@@ -5,179 +5,168 @@ weight: 3
+By default, Dedicated projects include a [Fastly](./fastly.md) CDN managed by Platform.sh.
+Self-service Grid plans don't include a CDN by default, but you can set up one at any time.
+
+## CNAME records
+
+To start routing client traffic through your CDN,
+create `CNAME` records for your domain names through your DNS provider.
+For more information, see you DNS provider's official documentation.
+
+Note that `CNAME` records can't point to apex domains,
+but most CDN providers offer [workarounds](../steps/dns.md#handling-apex-domains).
+
+## DDoS protection
+
+In a distributed denial of service (DDoS) attack, 
+an attacker floods a server with fake requests.
+The resources hosted by that server become inaccessible to real users.
+
+All sites hosted by Platform.sh live on infrastructure provided by major cloud providers.
+By default, you benefit from the powerful DDoS protection services these cloud providers offer.
+In most cases, these protection services are enough to keep your site safe from DDoS attacks,
+including when you use a CDN.
+
+Enterprise and Elite projects on Platform.sh also include a [web application firewall (WAF)](../../security/waf.md). 
+Depending on your needs, you can install your own WAF and add security measures not included in your plan.  
+
+## Third-party TLS certificates
+
+Dedicated plans include two [transport layer security (TLS) certificates](../../other/glossary.md#transport-layer-security-tls), 
+an apex and a wildcard one.
+This allows for encryption of all traffic between your users and your app. 
+
+If you want to, you can also provide your own third-party TLS certificate.
+[Transfer your certificate](../../development/file-transfer.md), 
+its unencrypted private key and the intermediate certificate to a non-web accessible [mount](../../create-apps/app-reference.md#mounts), 
+on an environment that only Platform.sh support and trusted users can access.
+In this way, your private key can't be compromised.
+To add the TLS certificate to your CDN configuration, 
+[create a support ticket](../../overview/get-support.md#create-a-support-ticket).
+
+Note that when you add your own third-party TLS certificate,
+you are responsible for renewing them in due time.
+Failure to do so may result in outages and compromised security for your site.
+
+If you need an Extended Validation TLS certificate, you can get it from any TLS provider. 
+To add it to your CDN configuration, [create a support ticket](../../overview/get-support.md#create-a-support-ticket).
+
+## Host header forwarding
+
+When a HTTP request is made to a website, the client adds a `Host` header to the request. 
+The value of this header is the domain name the request is made to. 
+When a server hosts multiple websites, like what a CDN does,
+it can use the `Host` header to identify which domain to access to handle the request.
+
+When a request is made from a client for an object on a CDN edge server, 
```

Comment:
:pick: I feel like the `object` here isn't completely clear. At least, it made be confused at first. I wasn't sure what an `object` was in this context since it's not used elsewhere here. Not sure if there's something better though. :shrug:

Reply:
Yes, `resource` was clearer for me.